### DISCUSSION TITLE
docker image for version v22.3.2.2-lts

### DISCUSSION THREAD

**Discussion by gauravphoenix** [ClickHouse/ClickHouse#35389] [Q&A] [answered]
I can't seem to find that docker image for the latest release `v22.3.2.2-lts`,  usually the docker hub gets updated with the release showing up soon after the github packages. 
maybe something is broken?

 cc @Felixoid

**Comment by Felixoid** [comment, marked as answer]
CI is in progress in https://github.com/ClickHouse/ClickHouse/pull/35211

I am pushing the normal and alpine repo RN https://github.com/ClickHouse/ClickHouse/pull/35211#issuecomment-1071795941

**Reply by mattvb91** [reply]
https://registry.hub.docker.com/r/yandex/clickhouse-server/tags am i looking in the right place or are the latest releases missing?

**Reply by Felixoid** [reply]
It's the wrong user name. https://registry.hub.docker.com/r/clickhouse/clickhouse-server/tags?page=1&name=la
